src/containers/shared/components/Amount.tsx

```diff
@@ -3,15 +3,10 @@ import { CURRENCY_OPTIONS, XRP_BASE } from '../transactionUtils'
 import { useLanguage } from '../hooks'
 import { localizeNumber } from '../utils'
 import Currency from './Currency'
+import { ExplorerAmount } from '../types'
 
 export interface AmountProps {
-  value:
-    | string
-    | {
-        issuer?: string
-        currency: string
-        amount: number
-      }
+  value: ExplorerAmount
   displayIssuer?: boolean
 }
```

Comment:
Is the amount object going to link to the token instead of the issuer?

Reply:
That is on the road map.  It will be a change to the Currency component.

Reply:
Went ahead and created the PR #384